eox_tenant/edxapp_wrapper/enrollments.py

```diff
@@ -0,0 +1,16 @@
+""" Backend abstraction. """
```

Comment:
@cocococosti I think is better for clarity and future code maintenance that this docstring is more specific as to what backend is referring to.